src/Microsoft.VisualStudio.AppDesigner/PropPages/PropertyControlData.vb

```diff
@@ -265,10 +265,16 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         '''  are looked up by name (not DISPID).  This allows project flavors to hide one property and add another with the same name but
         '''  a different DISPID in order to "override" a property.
         ''' </summary>
-        Public ReadOnly Property PropertyName As String
+        Public Property PropertyName As String
             Get
                 Return _propertyName
             End Get
+            Protected Set(value As String)
+#If DEBUG Then
+                Debug.Assert(Not _isInitialized)
+#End If
```

Comment:
```suggestion
                Debug.Assert(Not _isInitialized)
```

The [`System.Diagnostics.Debug.Assert`](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.debug.assert?view=netcore-3.1) method has attribute `[Conditional("DEBUG")]`. We don't need another check in code here.

Reply:
The pipeline is hitting an error here, saying `BC30451: '_isInitialized' is not declared. It may be inaccessible due to its protection level`. The other parts of the code that use it are wrapped with the debug checks, so I'm wondering if that's why it's breaking.